repo/db/db.go

```diff
@@ -187,6 +193,7 @@ func initDatabaseTables(db *sql.DB, pin string) error {
 	if pin != "" {
 		sqlStmt = "PRAGMA key = '" + pin + "';"
 	}
+	// TODO: also add field to cafe_clients to link to cafe_dev_tokens
```

Comment:
Haven't done this yet... but maybe we want it now to make migration easier?

Reply:
i'd say this should be part of this PR, and put it into the same migration